src/Data/Uniques/jewel.lua

```diff
@@ -247,10 +247,12 @@ Variant: Pre 3.0.0
 Variant: Pre 3.10.0
 Variant: Current - Crit Chance
 Variant: Current - Min Endurance Charge
+Variant: Current - Min Power Charge
 {variant:1}Gain 15 Mana per Grand Spectrum
 {variant:2}Gain 30 Mana per Grand Spectrum
 {variant:3}25% increased Critical Strike Chance per Grand Spectrum
 {variant:4}+1 to Minimum Endurance Charges per Grand Spectrum
```

Comment:
The min endurance charge on is the crimson base, could you move htis one? 
![image](https://user-images.githubusercontent.com/17045310/185306706-727d25d7-6970-4d27-a428-df76618c7a03.png)